### ISSUE TITLE
Toastify: Pausing or extending React-Toastify Notifications

### ISSUE THREAD

**Issue by bpmcneilly** [benetech/MathShare#957]
The Toastify notifications provided to users have an automatic timeout associated with them, after which they disappear. Currently, the only method to pause or prevent the notifications from being removed from the page is by hovering a mouse over the notification. Some users may not be able to read text within the span of time currently provided, and others may not realize they can hover over the message to pause the timer. For users who do not use a mouse, there will be no method to pause these notifications.

#### Suggested Solution
The React-Toastify component provides a property `autoClose` that allows developers to specify how long a notification appears for. Additionally, the library provides the ability to set `autoClose` to `false`, which will require users to manually dismiss the notifications (though, this would require adding a button with the `closeToast` function to allow notifications to be dismissed from the keyboard). 

A `select` element could be provided within the Personalization Settings that will allow users to set how long they would like these tooltips to appear for.

**Comment by johnhbenetech** [comment]
Hey @alexrcabral @benetechMatt please suggest an functionality based on the suggestions above

If I don't receive a suggestion by Monday, I will make a decision and move forward

**Comment by johnhbenetech** [comment]
@rupeshparab Let's add a personalization setting to allow for user provided timeouts (in seconds). If not set default should be 10s as it is now.
